course_contents/1_intro/notes/03. Comparison Operators.md

```diff
@@ -0,0 +1,48 @@
+Each comparison operator returns True or False
+### Equal `==`
+
+`4 == 4  # True`
+
+`4 == 5 # False`
+
+- Can be used to check the truth value of an operand:
+
+`4 == True  #  True`
+
+`4 == False  # False`
+
+`0 == False  # True`
+
+- Can be used to compare two strings or any other data type:
+
+`"Hello" == "Hello"  # True`
+
+`"Hello" == "HELLO"  # False`
+
+`(100, 50) == 50  # False`
+
+`(100, 50) == (500, 219)  # False`
+
+### Not equal `!=`
+- The opposite of equal
+
+`4 != 4  # False`
+
+`4 != 5  # True`
+
+### Greater than `>`
+`10 > 10  # False`
+
+- Two string can be compared as such, based on their length:
+
+`"Random longer string" > "Shorter string"  # False`
```

Comment:
I think I would remove this 🤔  don't think it's very useful, and it's very specific to strings (have you ever done this in an actual program?)

Reply:
You're right. No, I haven't done this.